src/main/java/com/alkemy/wallet/service/impl/TransactionsServiceImpl.java

```diff
@@ -35,12 +39,27 @@ public List<TransactionDto> getByAccountAndType(Long accountId, String type) {
     return dtoList;
   }
 
+  @Override
+  public TransactionDto createTransaction(TransactionDto dto) {
+    TransactionEntity transactionEntity = transactionMap.transactionDto2Entity(dto);
+
+    transactionEntity.setAmount(dto.getAmount());
+    transactionEntity.setType(dto.getType());
+    transactionEntity.setAccountId(accountRepository.getById(dto.getAccountDto().getId()));
+    transactionEntity.setDescription(dto.getDescription());
+
+    ITransactionRepository.save(transactionEntity);
+
+    return dto;
+  }
+
   @Override
   public List<TransactionDto> transactionsById(Long userId) {
-    UserDto user=userService.findById(userId);
-    List<TransactionDto> dtoList=null;
-    for (int i=0;i<user.getAccounts().size();i++)
+    UserDto user = userService.findById(userId);
+    List<TransactionDto> dtoList = null;
+    for (int i = 0; i < user.getAccounts().size(); i++) {
```

Comment:
fijate si te sale el for each que se usa mas, queda mas lindo y es mas funcional. En este caso es lo mismo, pero cuando empezas a usar el otro no usas mas este (solo a modo de comentario)